addon/templates/components/tf-checkbox.hbs

```diff
@@ -0,0 +1,24 @@
+{{#if hasBlock}}
```

Comment:
Just for sake of "dryness", we _could_ use contextual components for those `checkbox-input`s. For example, if we used `ember-let`, this template could look like:

```hbs
{{let checkbox=(component 'checkbox-input' 
  class=class
  id=id
  disabled=disabled
  name=name
  value=value
  checked=checked
  click=(action "onClick")
  shapeStyle=shapeStyle
)}}

{{#if hasBlock}}
  <label class="tf-checkbox__label {{if checked 'tf-checkbox__label--checked'}} {{labelClass}}" for={{id}}>
    {{component checkbox}}
  </label>
{{else}}
  {{component checkbox}}
{{/if}}
```